src/NServiceBus.Core/Receiving/ReceiveConfiguration.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace NServiceBus
+{
+    using System;
+    using System.Threading.Tasks;
+    using ObjectBuilder;
+    using Transport;
+
+    class ReceiveConfiguration
+    {
+        public ReceiveConfiguration(LogicalAddress logicalAddress, string queueNameBase, string localAddress, string instanceSpecificQueue, TransportTransactionMode transactionMode, PushRuntimeSettings pushRuntimeSettings, bool purgeOnStartup, bool isEnabled)
+        {
+            LogicalAddress = logicalAddress;
+            QueueNameBase = queueNameBase;
+            LocalAddress = localAddress;
+            InstanceSpecificQueue = instanceSpecificQueue;
+            TransactionMode = transactionMode;
+            PushRuntimeSettings = pushRuntimeSettings;
+            IsEnabled = isEnabled;
+            PurgeOnStartup = purgeOnStartup;
+
+            SatelliteDefinitions = new SatelliteDefinitions();
+        }
+
+        public LogicalAddress LogicalAddress { get; }
+
+        public string LocalAddress { get; }
+
+        public string InstanceSpecificQueue { get; }
+
+        public TransportTransactionMode TransactionMode { get; }
+
+        public PushRuntimeSettings PushRuntimeSettings { get; }
+
+        public string QueueNameBase { get; }
+
+        public bool IsEnabled { get; }
+
+        public SatelliteDefinitions SatelliteDefinitions { get; }
```

Comment:
It seems like `SatelliteDefinitions` can be replaced by a List.

Reply:
yup, removed